Poker.BE/Poker.BE.Domain/Game/GameConfig.cs

```diff
@@ -7,32 +7,199 @@ namespace Poker.BE.Domain.Game
     public class GameConfig
     {
 
+        #region Fields
+        private int _maxNumberOfActivePlayers;
+        private int _maxNumberOfPlayers;
```

Comment:
No. Min/max numbers are for room configuration. Active Players is a property in Room class. Updated according to room state